devbox/provision/__main__.py

```diff
@@ -0,0 +1,51 @@
+# Pants would resolve this import correctly for us, but since we run this
+# directly through Pulumi sans-Pants we need this grossness
```

Comment:
I spent about 3 hours mucking with making Pants, Pulumi and `make typecheck` getting everything happy module wise. This is slightly different than the other repos, in fact! 

in `pulumi/grapl`, we reference `infra` as a top-level module (since `pulumi/` has no __init__.py)
![image](https://user-images.githubusercontent.com/69007229/163032684-8c4b4c38-a6fc-410b-9799-ab26b00dc03b.png)

This was actually a good canary-in-the-coal-mine as my `infra` folder was _also_ a top-level module and as such was overriding the previous `pulumi/` one.

Here, `provision/` has an `__init__.py` in it meaning the top level module is `provision`.
![image](https://user-images.githubusercontent.com/69007229/163032855-9dcbc36f-dde5-4eee-943a-48360f31f157.png)

Sidenote: Don't try adding a `__init__.py` to `pulumi/` or, whoops, you'll have overwritten the pypi Pulumi module.